storage/storage.go

```diff
@@ -120,10 +124,19 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	// need to account for STORAGE_EMULATOR_HOST override when setting the default endpoints.
 	if host := os.Getenv("STORAGE_EMULATOR_HOST"); host == "" {
 		// Prepend default options to avoid overriding options passed by the user.
-		opts = append([]option.ClientOption{option.WithScopes(ScopeFullControl), option.WithUserAgent(userAgent)}, opts...)
+		opts = append([]option.ClientOption{option.WithScopes(ScopeFullControl, "https://www.googleapis.com/auth/cloud-platform"), option.WithUserAgent(userAgent)}, opts...)
```

Comment:
Could this create issues for users who don't have this scope enabled currently and are not trying to use Signed URL features?

Reply:
I don't think so, scopes work hand-in-hand with IAM for access to things. The scope just says it is allowed to talk, IAM says you may talk.

This does bring up a documentation point though. Users potentially have to enable the IAM credentials API if they are using compute creds and that flow.

Reply:
Yep for sure, we should definitely document that.